Story/Voltaire/CoreVoltaire.cs

```diff
@@ -0,0 +1,432 @@
+        Story.MapItemQuest(3110, "treasureisland", 1958, 1);
+        
+        //Captain Von Poach 3111
+        Story.KillQuest(3111, "treasureisland", "Captain Von Poach");
+    }
+
+    public void GustavduGrog()
+    {
+        if (Core.isCompletedBefore(3119))
+            return;
+
+        //Me Knickers Got a Big Hole Across th' Bum 3114
+        Story.KillQuest(3114, "skullpunch", new[] { "Vampirate", "Fishbones" });
+
+        //Where'd Me Ship's Cargo? 3115
+        Story.KillQuest(3115, "skullpunch", new[] { "Fishbones", "Fishbones", "Fishbones", "Fishbones" });
+
+        //A Most Important Package 3116
+        Story.KillQuest(3116, "skullpunch", "Fishbones");
+
+        //Comfort Food 3117
+        Story.KillQuest(3117, "skullpunch", new[] { "Shelleton", "Fishwing" });
+
+        //Full o' Holes 3118
+        Story.KillQuest(3118, "skullpunch", "Shelleton");
+
+        //Keelhaulin' a Kraken! 3119
+        Story.KillQuest(3119, "chaoskraken", "Chaos Kraken");
+    }
+
+    public void Wormhole()
+    {
+        //Something In The Air 5051
+        Story.KillQuest(5051, "wormhole", new[] { "Goth Girl", "Vamp Boy" });
+        
+        //They're Not Red Shirts - They're Deadshirts! 5052
+        Story.MapItemQuest(5052, "wormhole", new[] { 4420, 4421, 4422 });
+        Story.KillQuest(5052, "wormhole", "Deadshirt");
+
+        //Curses! 5053
+        Story.KillQuest(5053, "wormhole", "Cursed Alien");
+
+        //Computers Don't Lie 5054
+        Story.MapItemQuest(5054, "wormhole", new[] { 4423, 4424, 4425, 4426 });
+
+        //Beam Me Up! 5055
+        Story.KillQuest(5055, "wormhole", new[] { "Undead Astronaut", "Undead Astronaut" });
+
+        //Exotic Pets 5056
+        Story.KillQuest(5056, "wormhole", "Xenodog");
+
+        //Not Trobbles 5057
+        Story.KillQuest(5057, "wormhole", "Green Trobbolier|Purple Trobbolier"); // Be patient
+
+        //System Failure! 5058
+        Story.MapItemQuest(5058, "wormhole", 4428, 1);
+
+        //I Think I Saw This Movie Once 5059
+        Story.KillQuest(5059, "wormhole", "Space Horror");
+
+        //I Have A Short Fuse 5060
+        Story.MapItemQuest(5060, "wormhole", 4427, 6);
+        Story.KillQuest(5060, "wormhole", "Volatile Current");
+
+        //More Trobboliers 5061
+        Story.KillQuest(5061, "wormhole", "Green Trobbolier|Purple Trobbolier"); // Be patient
+
+        //Maybe These Guys Know Something 5062
+        Story.KillQuest(5062, "wormhole", new[] { "Stormslasher", "Undead Space Marine" });
+
+        //Look At That Awesome Armor! 5063
+        Story.KillQuest(5063, "wormhole", "Stormslasher");
+
+        //Guardians of the Warehouse 5064
+        Story.KillQuest(5064, "wormhole", "Guardian");
+
+        //Undead Space 5065
+        Story.KillQuest(5065, "wormhole", new[] { "Space Ghost", "Singularity" });
+
+        //Trobbolegion! 5066
+        Story.KillQuest(5066, "wormhole", "Trobbolegion");
+    }
+
+    public void Crownsreach()
```

Comment:
Name the map according to it's actuall name if all quests are in CrownsreachFXIII then the method name should be
CrownsReachFXIII